### PR TITLE
adding CSV export to members page

### PR DESCRIPTION
**Pull request by julien51**
# Description

This adds a button which, when clicked on will download the list of members on that page as CSV.
The download includes the same content that is visible on the page.

# Issues

Refs #5354

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

File: unlock-app/src/components/interface/MetadataTable.tsx [hunk 1]
```diff
@@ -33,34 +36,82 @@ export function camelCaseToTitle(s: string): string {
   )
 }
 
+/**
+ * Downloads a file with the key metadata as CSV
+ * Includes the colum name in the first row
+ */
+function downloadAsCSV(columns: any, metadata: any) {
```
Comment (cnasc) [inline unlock-app/src/components/interface/MetadataTable.tsx]: I think we'll want thorough tests on this to make sure we don't accidentally send corrupt data!

**Comment by benwerd** [comment]
Pardon the drive-by, but: this is really cool!

File: unlock-app/src/components/interface/MetadataTable.tsx (hunk 1 above)
Comment (julien51) [reply unlock-app/src/components/interface/MetadataTable.tsx]: done!

**Pull request merged**